editoast/src/views/timetable/stdcm.rs

```diff
@@ -115,13 +125,21 @@ pub(in crate::views) struct InfraIdQueryParam {
 /// If the simulation fails, the function uses a virtual train to detect conflicts
 /// with existing train schedules. It then returns both the conflict information
 /// and the pathfinding result from the virtual train's simulation.
+#[tracing::instrument(
+    target = "editoast::timetable",
+    name = "stdcm",
+    skip_all,
+    err,
+    fields(request)
+)]
 #[editoast_derive::route]
 #[utoipa::path(
     post, path = "",
     tag = "stdcm",
     request_body = inline(Request),
     params(("infra" = i64, Query, description = "The infra id"),
         ("id" = i64, Path, description = "timetable_id"),
+        ("return_debug_payloads" = Option<bool>, Path, description = "if true, extra payloads are returned to help with debugging"),
     ),
```

Comment:
suggestion:

```suggestion
   params(
       	("id" = i64, Path, description = "timetable_id"),
   		StdcmQueryParams
   ),
```

doc comments will be turned into OpenAPI descriptions.

Reply:
It turns them from query parameters into path parameters, I couldn't find a way to make it work that way while keeping them as query params :confused:

Reply:
Fixed in https://github.com/OpenRailAssociation/osrd/pull/13078/commits/285f85fa1f9dbac466a27efcd49d96dfdd90b7c0

The order changed in the openapi and generated ts files, but hopefully that doesn't change anything